src/Config/RuntimeConfigLoader.cs

```diff
@@ -61,6 +89,9 @@ protected void SendEventNotification(string message = "")
         OnConfigChangedEvent(new HotReloadEventArgs(MYSQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED, message));
         OnConfigChangedEvent(new HotReloadEventArgs(POSTGRESQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED, message));
         OnConfigChangedEvent(new HotReloadEventArgs(DOCUMENTOR_ON_CONFIG_CHANGED, message));
+
```

Comment:
nit: rename to indicate this function also handles signaling the change token. Currently, it says send event notification only.

Reply:
updating to `SignalConfigChanged` from `SendEventNotification`